packages/shared/src/components/cards/WelcomePostCardFooter.tsx

```diff
@@ -1,4 +1,5 @@
-import React, { ReactElement } from 'react';
+import React, { ReactElement, useRef } from 'react';
+import { sanitize } from 'dompurify';
```

Comment:
Just for a completeness, have you checked for possible alternatives with smaller bundle size? This one is 7kb gzipped which should be fine if it tree shakes.

Reply:
I believe tree shaking works on this package. Though I was not present when this lib was introduced and had no history about the other options.

Reply:
Oh sorry, thought it was introduced in this PR. Ignore.